### PR TITLE
RMET-2823 Local Notifications Plugin - Replace Android Support with AndroidX

### PR DESCRIPTION
**Pull request by alexgerardojacinto**
## Description
- Replaced the dependency to android's Support Library with its AndroidX counterpart.

## Context
https://outsystemsrd.atlassian.net/browse/RMET-2823

## Type of changes

- [ ] Fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Refactor (cosmetic changes)
- [ ] Breaking change (change that would cause existing functionality to not work as expected)

## Platforms affected
- [X] Android
- [ ] iOS
- [ ] JavaScript

## Tests

- Tested on MABS 10 and MABS 9.
- Note: MABS 10 builds running on Android 14 still need the fix that will be made in https://outsystemsrd.atlassian.net/browse/RMET-2666, but that fix is out of the scope of this PR.

## Checklist

- [X] Pull request title follows the format `RNMT-XXXX <title>`
- [x] Code follows code style of this project
- [X] CHANGELOG.md file is correctly updated
- [ ] Changes require an update to the documentation
	- [ ] Documentation has been updated accordingly

### CODE REVIEW

File: src/android/build/localnotification.gradle [hunk 1]
```diff
@@ -26,21 +26,7 @@ if (!project.ext.has('appShortcutBadgerVersion')) {
...
 }
 
-String getVersion() {
-  def compileSdkVersion = Integer.parseInt(project.android.compileSdkVersion.split("-")[1])
-  return compileSdkVersion >= 28 ? "28.0.0" : "26.+" 
-}
-
 dependencies {
     implementation "me.leolin:ShortcutBadger:${appShortcutBadgerVersion}@aar"
+    implementation 'androidx.media:media:1.6.0'
```
Comment (hectorjdgdev) [inline src/android/build/localnotification.gradle:31]: Is there no need of update also this
import android.support.v4
in the
OutSystems/cordova-plugin-local-notifications/src/android/notification/Options.java
and in 
OutSystems/cordova-plugin-local-notifications/src/android/notification/Builder.java

**Review by hectorjdgdev — CHANGES_REQUESTED** [review]

File: src/android/build/localnotification.gradle (hunk 1 above)
Comment (hectorjdgdev) [reply src/android/build/localnotification.gradle:31]: I saw it, is in the other PR

Comment (alexgerardojacinto) [reply src/android/build/localnotification.gradle:31]: I also thought we had to update those when I created the task, but it turns out that there is no import starting with `androidx` that contains the `MediaSessionCompat` class, which is needed both in `Options.java` and `Builder.java`. 

However, by removing the dependency on the Support Library from the Gradle file, MABS 10 builds work. If we keep them, MABS 10 builds fail. So I believe we only need to remove them from the Gradle file.

Comment (hectorjdgdev) [reply src/android/build/localnotification.gradle:31]: Yes make sense, MediaSessionCompact belongs to androidx also, [x/platform/frameworks/support/+/androidx-main:media/media/src/main/java/android/support/v4/media/session/MediaSessionCompat.java?q=file:android%2Fsupport%2Fv4%2Fmedia%2Fsession%2FMediaSessionCompat.java%20class:android.support.v4.media.session.MediaSessionCompat](url)

**Review by hectorjdgdev — APPROVED** [review]

File: src/android/build/localnotification.gradle (hunk 1 above)
Comment (alexgerardojacinto) [reply src/android/build/localnotification.gradle:31]: Yes exactly.

**Review by OS-martacarlos — APPROVED** [review]

**Pull request merged**